dockerfiles/alpine/analytics-dashboard/README.md

```diff
@@ -0,0 +1,65 @@
+# Dockerfile for WSO2 Micro Integrator #
+
+This section defines the step-by-step instructions to build an [Alpine](https://hub.docker.com/_/alpine/) Linux based
+ Docker image for WSO2 Analytics Dashboard 1.0.0.
+
+## Prerequisites
+
+* [Docker](https://www.docker.com/get-docker) v17.09.0 or above
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) client
+
+## How to build an image and run
+
+##### 1. Checkout this repository into your local machine using the following Git client command.
+
+```
+git clone https://github.com/wso2/docker-ei.git
+```
+
+>The local copy of the `dockerfiles/alpine/analytics-dashboard` directory will be referred to as `ANALYTICS_DASHBOARD_DOCKERFILE_HOME` from this point onwards.
+
+##### 2. Build the Docker image.
+
+- Navigate to `<ANALYTICS_DASHBOARD_DOCKERFILE_HOME>` directory. <br>
+  Execute `docker build` command as shown below.
+    + `docker build -t wso2ad:1.0.0-alpine .`
```

Comment:
@ThilinaManamgoda are we planning to name this repository as `ad`? IMO, better to use the complete term `analytics-dashboard`.

Reply:
IMO, we can name it as `wso2mi-monitoring-dashboard`. WDYT?